packages/schematics/angular/resolver/schema.json

```
@@ -25,6 +25,11 @@
       "description": "When true (the default), creates the new files at the top level of the current project.",
       "default": true
     },
+    "functional": {
+      "type": "boolean",
+      "description": "When true, creates the resolver as a `ResolveFn`.",
```

NIT: we are trying to avoid when true and when false. As this can be confusing when used as a CLI flag

```suggestion
      "description": "Creates the resolver as a `ResolveFn`.",
```

---

👍 Sounds good!